README.md

```diff
@@ -1,7 +1,15 @@
 # House of Games API
 
-## Initial Setup
+## **Initial Setup**
 
-### .env files
+### **.env files**
 
-You will need to create two .env files in order to successfully connect the two databases locally: .env.test and .env.development. Into each, add add `PGDATABASE=<database_name_here>`, with the correct database name for that environment. Please reference the /db/setup.sql file for the database names.
+You will need to create two .env files in order to successfully connect the two databases locally: .env.test and .env.development. 
+
+### *Creating the .env files*
```

Comment:
Love the detail here, I think its safe to assume that anyone looking at your README will have some level of VSCode literacy. I think you can simplify this section by just explaining the format the files names will need to be.

Reply:
Thanks Haz! This has been done.